client/my-sites/site-settings/cloudflare.js

```diff
@@ -30,32 +40,76 @@ const Cloudflare = () => {
 		<>
 			{ showCloudflare && (
 				<>
-					<SettingsSectionHeader title={ translate( 'Cloudflare CDN' ) } />
-					<Card>
+					{ ! sitePlan && <QuerySitePlans siteId={ siteId } /> }
+					<SettingsSectionHeader title={ translate( 'CDN' ) } />
+					<CompactCard>
 						<div className="site-settings__cloudflare">
+							<div className="site-settings__cloudflare-illustration">
+								<img src={ jetpackIllustration } alt="" />
+							</div>
 							<div className="site-settings__cloudflare-text">
+								<p className="site-settings__cloudflare-title">
+									{ translate( 'Jetpack Site Accelerator' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( 'Site Accelerator' )` *ES Score: 6*